api/api.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+import (
+	"golang-interview-exercise/api/block"
+	"golang-interview-exercise/api/subscribe"
+	"golang-interview-exercise/api/transactions"
+
+	"github.com/labstack/echo/v4"
+)
+
+func RegisterAPI(e *echo.Echo) error {
+	e.GET("/block", block.GetCurrentBlock)
+	e.POST("/subscribe", subscribe.PostSubscribe)
+	e.GET("/transaction/:address", transactions.GetTransaction)
+
+	return nil
+}
```

Comment:
Em đọc code thetan-world, mình follow 3 layers: controller -> services -> data (repository).
- Controller: xử lý nhận request, validate request, gọi service, trả http response.
- Service: xử lý business logic và gọi data để thay đổi state
- Data (repository): chỉ xử lý data và đảm bảo tính toàn vẹn của fields nếu có, không có business logic.